wordpress-mobile/gutenberg-mobile
# VideoPress block: Initial setting tests

Part of the coverage efforts outlined in https://github.com/wordpress-mobile/gutenberg-mobile/issues/5734

## Description

The changes in this PR introduce tests that ensure the VideoPress block's title, description, and Playback settings can be successfully updated, as well as some of the settings in the Playback Bar Color and Privacy and Rating panels.

_Note, the following are not covered by this PR, and will be worked on in separate PRs:_

* Rating and Privacy picker selection.
* Various color selections for Playback Bar Color.
* Setting synchronization between the app and the web. 
 
## To test:

* Verify the HTML in [the snapshot file](https://github.com/wordpress-mobile/gutenberg-mobile/blob/f8736c8dd5a706deb525308e8a5e637e6804a331/src/test/videopress/__snapshots__/edit.js.snap) matches with the reality of what the block's HTML would look after making the relevant changes.
* Run the `TEST_RN_PLATFORM=ios npm run test src/test/videopress/edit.js` and `TEST_RN_PLATFORM=android npm run test src/test/videopress/edit.js` commands from the terminal to verify the tests pass as expected. Also, verify the tests on this PR pass.

<hr />

PR submission checklist:
